air/src/lib.rs

```diff
@@ -144,6 +145,9 @@ impl Air for ProcessorAir {
         // Add the range checker's assertions for the last step.
         range::get_assertions_last_step(&mut result, last_step);
 
+        // Add assertions for the aux table
+        aux_table::get_assertions(&mut result);
+
```

Comment:
Two notes here:
1. As mentioned above, I would prefer to use `aux_table::get_assertions_first_step` instead.
2. I would prefer to move this up so it happens before we start setting assertions for the last step (above the "set assertions for the last step" divider comment)